LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift

```diff
@@ -316,14 +324,26 @@ private struct CorrectionRangeOverridesGuardrailWarning: View {
     private func singularWarningTitle(for threshold: SafetyClassification.Threshold) -> Text {
         switch threshold {
         case .minimum, .belowRecommended:
-            return Text("Low Correction Value", comment: "Title text for the low correction value warning")
+            if crossedThresholds.keys.contains(.preMeal) {
```

Comment:
Consider passing which override the warning applies to into `singularWarningTitle` rather than getting it from `crossedThresholds`.